src/app/designationEditor/photoModal/photo.modal.js

```diff
@@ -1,28 +1,39 @@
 import angular from 'angular';
 import 'angular-upload';
 
+import designationEditorService from 'common/services/api/designationEditor.service';
+
 let controllerName = 'photoCtrl';
 
 class ModalInstanceCtrl {
 
   /* @ngInject */
-  constructor( designationNumber, campaignPage, photos, photoLocation, selectedPhoto ) {
+  constructor( $timeout, designationNumber, campaignPage, photos, photoLocation, selectedPhoto, designationEditorService ) {
+    this.$timeout = $timeout;
+    this.designationEditorService = designationEditorService;
+
     this.designationNumber = designationNumber;
     this.campaignPage = campaignPage;
     this.photoLocation = photoLocation;
     this.selectedPhoto = selectedPhoto;
     this.photos = photos;
   }
 
-  uploadComplete(response) {
-    this.photos = response.data;
-    this.uploading = false;
+  uploadComplete() {
+    //refresh photos (set timeout to give Adobe time to add to DAM)
+    this.$timeout(() => {
+      this.designationEditorService.getPhotos(this.designationNumber, this.campaignPage).then((response) => {
```

Comment:
@adammeyer is this method sending the campaign code to the endpoint? image upload and image listing (POST and GET) does not process the campaign parameter (as we previously check yesterday, images are shared between a designation and their campaigns)